src/Resources/contao/library/Contao/Widget.php

```diff
@@ -83,6 +84,7 @@
  * @property string                  $slabel            The submit button label
  * @property boolean                 $preserveTags      Preserve HTML tags
  * @property boolean                 $decodeEntities    Decode HTML entities
+ * @property boolean                 useRawRequestData  Use the raw request data from the Symfony request
```

Comment:
Should we add `useRawRequestData` to the switch-case on [line 345](https://github.com/Toflar/core-bundle/blob/509805560c0c5753138fbd2cacad8cad21ed62a1/src/Resources/contao/library/Contao/Widget.php#L345) too?